shell/layouts/default.vue

```diff
@@ -307,6 +311,14 @@ export default {
   mounted() {
     // Sync the navigation tree on fresh load
     this.$nextTick(() => this.syncNav());
+
+    this.wmPin = window.localStorage.getItem('wm-pin') || BOTTOM;
+
+    // two-way binding this.wmPin <-> draggableZone.pin
+    this.$refs.draggableZone.pin = this.wmPin;
+    this.$watch('$refs.draggableZone.pin', (pin) => {
```

Comment:
This watcher should be destroyed if we move out of the `default` layout.

Check example here: https://github.com/rancher/dashboard/blob/master/shell/components/fleet/ForceDirectedTreeChart/index.vue#L328-L341

Reply:
Right, fixed!